Point_set_processing_3/examples/Point_set_processing_3/write_ply_points_example.cpp

```diff
@@ -32,7 +32,7 @@ namespace CGAL {
       if (is_ascii(out))
         out << int(c[0]) << " " << int(c[1]) << " " << int(c[2]) << " " << int(c[3]);
       else
-        CGAL::write(out, c);
+        out.write(reinterpret_cast<const char*>(&c), sizeof(c));
```

Comment:
Actually it isn't: there's a tag `Io_tag()` that's deduce. If it is `io_Read_write()`, it does the `reinterpret_cast` but apparently, in that case, it is `io_Operator`: as a result, it calls `oformat()`… which calls this method, which calls `oformat()`, etc. Hence the bug.

I'm not sure how these tags are supposed to be handled.